aztk/node_scripts/setup_host.sh

```diff
@@ -12,22 +12,32 @@ container_name=$1
 docker_repo_name=$2
 
 echo "Installing pre-reqs"
-apt-get -y update
-apt-get install -y --no-install-recommends linux-image-extra-$(uname -r) linux-image-extra-virtual
-apt-get install -y --no-install-recommends apt-transport-https
-apt-get install -y --no-install-recommends curl
-apt-get install -y --no-install-recommends ca-certificates
-apt-get install -y --no-install-recommends software-properties-common
-apt-get install -y --no-install-recommends python3-pip python3-venv python-dev build-essential libssl-dev
-echo "Done installing pre-reqs"
-
-# Install docker
-echo "Installing Docker"
-curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
-apt-get -y update
-apt-get -y install docker-ce
-echo "Done installing Docker"
+time(
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
```

Comment:
we could maybe split those into different files so it makes it more readable

Reply:
split what into different files? the inside of the time statement?

Reply:
Yeah